docs.html

```diff
@@ -106,7 +106,7 @@ <h2>Button</h2>
                 <a href="" class="btn btn-primary-link">Primary</a>
                 <a href=""><i class="btn-icon btn-primary-icon fas fa-shopping-cart"></i></a>
                 <a href="" class="btn btn-primary-icon-text"><i class="btn-icon fas fa-shopping-cart"></i>Add to cart</a>
-                <a class="btn-float-action"><i class="fas fa-plus"></i></a>
+                <a href="" class="btn-float-action"><i class="fas fa-plus"></i></a>
```

Comment:
dont leave href="". If you doesnt want anything just put # inside href tag.

Reply:
Will do this